### PR TITLE
Add scan command to run scanners

### PR DESCRIPTION
**Pull request by turbaszek**
Users can run scanners via `scan` command:
```
➜ kibble scan --help
Usage: kibble scan [OPTIONS]

Options:
  -t, --type TEXT     Specific type of scanner to run (default is run all
                      scanners)

  -e, --exclude TEXT  Specific type of scanner(s) to exclude
  -o, --org TEXT      The organisation to gather stats for. If left out, all
                      organisations will be scanned.

  -a, --age TEXT      Minimum age in hours before performing a new scan on an
                      already processed source. --age 12 will not process any
                      source that was processed less than 12 hours ago, but
                      will process new sources.

  -s, --source TEXT   Specific source (wildcard) to run scans on.
  -v, --view TEXT     Specific source view to scan (default is scan all
                      sources)

  --help              Show this message and exit.
```

```
➜ docker-compose -f docker-compose-dev.yaml run setup kibble scan
Creating kibble_setup_run ... done
Kibble Scanner starting
Using direct ElasticSearch broker model
Connecting to ElasticSearch database at http://elasticsearch:9200
Connected!
This is a type-less DB, expanding database names instead.
We're using ES >= 7.x, NO DOC_TYPE!
All done scanning for now, found 0 organisations and 0 sources to process.
```

### CODE REVIEW

File: kibble/cli/scanner_command.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        self.broker = broker
+        self.org = org
+        self.id = i
+        self.bit = self.broker.bitClass(self.broker, self.org, i)
+        self.stype = t
+        self.exclude = e
+        print("Initialized thread %i" % i)
+
+    def run(self):
+        from kibble.scanners import scanners
```
Comment (kaxil) [inline kibble/cli/scanner_command.py:61]: Is this here to delay import or causes an import cycle?

File: kibble/cli/scanner_command.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    scanners: List[str] = None,
+    exclude: List[str] = None,
+    org: str = None,
+    age: int = None,
+    source: str = None,
+    view: str = None,
+):
+    global PENDING_OBJECTS
+
+    print(f"Kibble Scanner starting")
```
Comment (kaxil) [inline kibble/cli/scanner_command.py]:
```suggestion
    print("Kibble Scanner starting")
```

File: kibble/cli/scanner_command.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    print("Using direct ElasticSearch broker model")
+    broker = kibbleES.Broker()
+
+    org_no = 0
+    source_no = 0
+    for org_item in broker.organisations():
+        if not org or org == org_item.id:
+            print("Processing organisation %s" % org_item.id)
```
Comment (kaxil) [inline kibble/cli/scanner_command.py]:
```suggestion
            print(f"Processing organisation {org_item.id}")
```

File: kibble/cli/scanner_command.py (hunk 1 above)
Comment (turbaszek) [reply kibble/cli/scanner_command.py:61]: To delay import, the `__init__` in `scanners` has top level logic which gets executed when importing the module. This is something that can be addressed later

Comment (kaxil) [reply kibble/cli/scanner_command.py:61]: +1

**Review by kaxil — APPROVED** [review]

**Pull request merged**